src/models/configurationSettings.ts

```diff
@@ -1,4 +1,4 @@
-"use strict";
+/// <reference path="../../typings/vscode-typings.d.ts" />

```

Comment:
I found that the typescript compiler will introduce "use strict" anyway.
Then I added the ref to the type definitions in order to get IntelliSense for the vscode API.